data/docs.yml

```diff
@@ -256,6 +256,12 @@
       description: The client implement for SkyWalking BanyanDB in Java
       user: apache
       repo: skywalking-banyandb-java-client
+      docs:
+        - version: latest
+          link: https://github.com/apache/skywalking-banyandb-java-client/blob/main/README.md
+        - version: v0.1.0
+          link: https://github.com/apache/skywalking-banyandb-java-client/blob/v0.1.0/README.md
+          commitId: ec763d30d7c8a1316adf9a1f5e0ab3c84b4e122b
```

Comment:
This is not correct. Take a look at https://github.com/apache/skywalking-website/blob/5b050d37967111b716715adb863c354a389330ed/data/docs.yml#L112

This is only working when you have a docs folder with website requirement.

Reply:
This folder https://github.com/apache/skywalking-banyandb/tree/main/docs with [menu.yml](https://github.com/apache/skywalking-banyandb/blob/main/docs/menu.yml) are required if you want to use a commit ID to manage docs. If you are going to use github link, use link only

Reply:
> This folder https://github.com/apache/skywalking-banyandb/tree/main/docs with [menu.yml](https://github.com/apache/skywalking-banyandb/blob/main/docs/menu.yml) are required if you want to use a commit ID to manage docs. If you are going to use github link, use link only

I see. Thanks for reminding